pages/studentInstanceQuestionHomework/studentInstanceQuestionHomework.ejs

```diff
@@ -51,6 +51,10 @@
                   <td><%- include('../partials/questionValue', {value: instance_question.current_value});  %></td>
                 </tr>
                 <tr>
+                  <td>Awarded:</td>
+                  <td><%- include('../partials/questionAwardedPoints', {awarded_points: instance_question.variants_points_list, init_points: assessment_question.init_points});  %></td>
+                </tr>
+                <tr>
                   <td>Points:</td>
```

Comment:
On Exams the total points line is called "Awarded points", which makes the use of "Awarded" and "Points" as separate items on Homeworks a bit confusing. How about calling the three lines on Homeworks:
* Value: 2
* Best submission: 45%
* Awarded points: 3.2/10

Reply:
I understand your concern here. Could we rename the three rows without changing their content? One thing I like about what is currently called `Awarded` is that it makes clear when students are stuck (i.e., it makes clear that they won't be able to accumulate more points until they get 100%).

(Also, would the value of `Best submission` vanish when a student gets 100%? It should, because at that point, the student is on the next variant, and so their best submission is again 0%. This seems a little confusing.)

Thoughts?